x/ibc/02-client/keeper/client.go

```diff
@@ -50,15 +50,27 @@ func (k Keeper) CreateClient(
 }
 
 // UpdateClient updates the consensus state and the state root from a provided header
-func (k Keeper) UpdateClient(ctx sdk.Context, clientID string, header exported.Header) error {
+func (k Keeper) UpdateClient(
```

Comment:
I think this function should be defined per client type, not in this file, see `verifyHeaderAndUpdateState`